evalml/automl/automl_algorithm/automl_algorithm.py

```diff
@@ -32,8 +32,8 @@ def __init__(self,
         self.max_iterations = max_iterations
         self._tuner_class = tuner_class or SKOptTuner
         self._tuners = {}
-        for p in self.allowed_pipelines:
-            self._tuners[p.name] = self._tuner_class(p.hyperparameters, random_seed=self.random_seed)
+        for pipeline in self.allowed_pipelines:
```

Comment:
No functional change, but I noticed it's hard to use `pdb` in this loop because `p` gets mistaken for a command. Can revert if confusing 😂

Reply:
Good call!